### DISCUSSION TITLE
BrieFiniteElement.Net in 3ds MAX.

### DISCUSSION THREAD

**Discussion by Torabi** [BriefFiniteElementNet/BriefFiniteElement.Net#119] [Ideas]
here is a short video showing the implementation of BrieFiniteElement.Net in 3ds MAX. this was done around 2016 but I never published it as I felt this repository wasn't active enough. I was looking for some old codes and I ended up here again. I saw that things are moving slowly so now I am thinking to publish this work as an open source plug-in for 3ds MAX users. I am also interested in collaboration and pushing things forward. Please enjoy : [PARA 3d FEM](https://youtu.be/ZHfw0jqlCgM)

**Comment by epsi1on** [comment]
Hello Dear Mr. Ali,
The plugin looks very nice, specially in visualization of deformed shape of model. BFE itself do not have such visualizer. 
Have you seen any inconsistency / bug in the library? if yes please make an issue...
Thanks in advance

**Reply by Torabi** [reply]
Thanks for your feedback. Surly I am interested in sending feedback, and probably collaborating on this project. I will be sending some pull request soon. My area of interest are the visualization, shell analysis and more importantly the finite element of objects made of planar sheets such as steel and wood. I think your library is very clear and easy to understand, however there are some concern regarding its extendibility. If you have some free I would like to get in touch and discuss the future of BFEM further.

**Comment by epsi1on** [comment]
Also please note that some other open source .NET FEM library can be found in the list here:
https://github.com/wo80/CSparse.NET/network/dependents?package_id=UGFja2FnZS0xNTY4MTk5NTY%3D

**Reply by Torabi** [reply]
I checked the list, this one looks more popular..
